helmfile/values/fluentd-wc.yaml.gotmpl

```diff
@@ -93,7 +93,6 @@ extraConfigMaps:
       tag kubeaudit.*
       read_from_head true
       skip_refresh_on_startup true
-      enable_watch_timer false
```

Comment:
Is this not needed anymore for fluentd?

Reply:
I had troubles getting logs for the Kubernetes index with it disabled. It would get some logs when it started, but afterwards it would basically stop sending logs.
Without the watch timer it uses ionotify to check for update which apparently does not work reliably for me.